### PR TITLE
Switching to the Paid for Fixer

### PR DESCRIPTION
**Pull request by vikkio88**
- handling error if fixer return `success:false` (happens only in paid version if currency not allowed in plan)
- throwing exception if currency not allowed from provider
- extending valid currencies to all

### CODE REVIEW

**Comment by mheki** [comment]
can we update tests also, please?

File: config/app.php [hunk 1]
```diff
@@ -35,5 +35,10 @@
         'fixer' => getenv('FIXER_KEY') ?? '',
         'currencyLayer' => getenv('CURRENCY_LAYER_KEY') ?? '',
     ],
-    'allowed_currencies' => ['GBP', 'USD', 'EUR', 'NOK'],
+    'allowed_currencies' => [
+        'AUD', 'BGN', 'BRL', 'CAD', 'CHF', 'CNY', 'CZK', 'DKK', 'EUR',
```
Comment (mheki) [inline config/app.php]: this will have to be a business decision which currencies we do allow

Comment (vikkio88) [reply config/app.php]: those are all the one that are accepted/given back from fixer

**Comment by vikkio88** [comment]
this branch now is ready to merge as the api key now allow us to switch the base currency to every currency allowed

**Comment by mheki** [comment]
🚢 it

**Pull request merged**